src/lib.rs

```diff
@@ -313,7 +314,12 @@ impl Language {
         if code.len() != 2 {
             return None;
         }
-        TWO_TO_THREE.get(code).cloned()
+
+        // SAFETY: Enum is repr usize and the codes are generated at compile-time
+        TWO_TO_THREE
+            .get(code)
+            .copied()
+            .map(|raw_lang| unsafe { mem::transmute(raw_lang) })
```

Comment:
I don't love the use of `unsafe` here. What alternatives have you considered?

Reply:
Not really anything. I just used the unsafe here since it's applicable and has the lowest overhead.

I guess we could call `Language::from_usize()`. That should work fine and get rid of the unsafe

Reply:
(small bonus: it would remove the need for the representation annotation)

Reply:
Removed the unsafe usage